tests/net45/FileTransferTest.cs

```diff
@@ -124,5 +133,48 @@ public async Task TestInitiateExportDocumentLineReport()
             var report = await Client.InitiateExportDocumentLineReportAsync(company.value[0].id, exportDocumentLine);
             Assert.NotNull(report);
         }
+
+        [Test]
+        public void TestUploadCertificateImage()
+        {
+            //Get the cert number. The account needs to have CertCapture 
+            //be provisioned, already have a certificate created, and the
+            //certificate needs to be valid.
+            var certs = Client.QueryCertificates(DefaultCompanyId, string.Empty, "valid EQ true", null, null, string.Empty).value;
+
+            var certId = certs.FirstOrDefault().id.Value;            
+
+            //Get an image.
+            using (WebClient webClient = new WebClient()) {
+                byte[] jpgByteArr = webClient.DownloadData("https://developer.avalara.com/public/images/blog/12000-juris.jpg");
+
+                FileResult fileResult = new FileResult()
+                {
+                    ContentType = "multipart/form-data",
+                    Filename = "test_cert_image.jpg",
+                    Data = jpgByteArr
+                };
+
+                //Send request.
+                var certuploadResult = Client.UploadCertificateImage(DefaultCompanyId, certId, fileResult);
+
+                //Response should be "OK"
+                Assert.True(string.Equals(certuploadResult, "\"OK\""));
+            }
```

Comment:
Shall we extend the test and making sure we can download the uploaded file?

Reply:
Good idea. I'll extend the test.